content/_guides/clients.md

```diff
@@ -40,7 +40,7 @@ Some commonly chosen graphical clients for personal computers are are
 [Textual](https://www.codeux.com/textual/) (MacOS).
 
 On phones,
-[Revolution](https://play.google.com/store/apps/details?id=io.mrarm.irc) is
+[Revolution](https://f-droid.org/de/packages/io.mrarm.irc/) is
```

Comment:
`/de/` makes the F-Droid website appear in German. s/de/en/?

Reply:
Wow good catch, it did that automatically 😓
Let me fix.

Reply:
Fixed the localization for Revolution, as well as for LimeChat lmao